packages/docs/concepts/menus.yaml

```diff
@@ -0,0 +1,156 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/general.yaml.njk
+  vars:
+    pageId: menus
+    pageTitle: Menus
+    section: Concepts
+    filePath: concepts/menus.yaml
+    content:
+      - id: menu_filter_alert
+        type: Alert
+        properties:
+          type: info
+          showIcon: false
+          message: The menu items are filtered according to pages the user is authorized to see. This can be controlled by making pages public or private, or by assigning roles to authenticated users. See <a href = "./protected-pages">protected pages</a> and <a href = "./roles">roles</a> for more on how to set this up.
+
+      - id: md1
+        type: MarkdownWithCode
+        properties:
+          content: |
+            Menu objects are objects with links to pages. They are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+
+            Blocks like `PageSiderMenu` use the menu with `id: default` by default. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured.
+
+            More than one menu can be configured in an app. As an example, this can be used when two logically different sections in the app need different menus.
+
+            If OpenID Connect authentication and role based authentication is configured, pages that the user is restricted from seeing are filtered from the menu object.
+
+            The menu objects can be obtained by using the [`_menu`](/_menu) operator.
+
+            The schema for a menu object is:
+
+            - `id: string`: __Required__ - A identifier for the menu. If it is `default`, it will be used as default by pages.
+            - `links: object[]`: An array of `MenuLink` or `MenuGroup` objects that form the links in the menu. `MenuGroups` can be two levels deep.
+
+            The schema for a `MenuLink` is:
+            - `id: string`: __Required__ - A identifier for the link unique to the menu.
+            - `type: string`: __Required__ - The type should be `MenuLink`.
+            - `pageId: string`: The id of the page to link to. Used as the title if no title is provided.
+            - `url: string`: An external url to link to.
+            -  `properties: object`: The properties of the menu link. These are:
+                - `title: string`: The title to display for the link.
+                - `icon: string | object`: The name of a [React Icon](https://react-icons.github.io/react-icons/search) or properties of an Icon block to use as the icon for the link. The icon is only shown if the link is at the top level of the menu (not in a menu group).
```

Comment:
~~The icon is only shown if the link is at the top level of the menu (not in a menu group).~~ 
I think this is not valid anymore, please check..

Reply:
Did the check. That statement isn't valid anymore.

Change made in 0f8e3ebcba1315a5f14cec4a02cd6f8426f948e2